bindings/java/src/test/java/org/openucx/jucx/UcpContextTest.java

```diff
@@ -40,6 +40,20 @@ public void testCreateUcpContextRdma() {
         UcpContext context = createContext(contextParams);
         closeContext(context);
     }
+
+    @Test
+    public void testConfigMap() {
+        UcpParams contextParams = new UcpParams().requestTagFeature()
+            .setConfig("LOG_LEVEL", "info").setConfig("ZCOPY_THRESH", "1");
+        UcpContext context = createContext(contextParams);
+        closeContext(context);
+
+        // Return back original config
+        contextParams = new UcpParams().requestTagFeature()
+            .setConfig("LOG_LEVEL", "warn").setConfig("ZCOPY_THRESH", "auto");
+        context = createContext(contextParams);
```

Comment:
is there some way to check programmatically the config was really applied?
maybe for some other field?